packages/formatting/presets/standard.yaml

```
@@ -0,0 +1,57 @@
+# @outfitter/formatting Standard Preset
+# Balanced formatting for most projects
+name: standard
+description: Balanced formatting rules suitable for most JavaScript/TypeScript projects
+
+common:
+  # Line length for wrapping
+  lineWidth: 80
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)